payments-core/src/main/java/com/stripe/android/payments/bankaccount/CollectBankAccountLauncher.kt

```diff
@@ -56,6 +79,42 @@ interface CollectBankAccountLauncher {
         fun create(
             fragment: Fragment,
             callback: (CollectBankAccountResult) -> Unit
+        ): CollectBankAccountLauncher {
+            return StripeCollectBankAccountLauncher(
+                fragment.registerForActivityResult(CollectBankAccountContract()) {
+                    callback(it.toExposedResult())
+                }
+            )
+        }
+
+        /**
+         * Create a [CollectBankAccountLauncher] instance with [ComponentActivity].
+         *
+         * This API registers an [ActivityResultLauncher] into the [ComponentActivity],  it needs
+         * to be called before the [ComponentActivity] is created.
+         */
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun createInternal(
+            activity: ComponentActivity,
```

Comment:
Nit: We can use `ActivityResultRegistryOwner` here instead. Then we don’t need the two methods.

Unfortunately too late for the public API.

Reply:
We would also need to unregister the callback, I think it is easier to keep it the way it is.